Sentinel-dVPN-node/deploy.yml

```diff
@@ -1,37 +1,39 @@
 ---
 version: "2.0"
 endpoints:
-  your_endpoint_name:  # # WARRING! SPECIFY UNIQ ENDPOINT NAME! ONLY LOWER CASE SYMBOLS!
+ uniq_name_endpoint: # unique name, for example "dvpn_dimokus:", and enter this name in line 23 and 28.
```

Comment:
add one space before `uniq_name_endpoint: ` line :-)

Reply:
Fix it